renku/core/management/migrations/models/v3.py

```diff
@@ -41,20 +41,24 @@ def __init__(self, **kwargs):
 class Person(Base):
     """Person migration model."""
 
+    client = None
+
     @classmethod
-    def from_git(cls, git):
+    def from_git(cls, git, client=None):
         """Create an instance from a Git repo."""
         name, email = get_user_info(git)
-        return cls(name=name, email=email)
+        instance = cls(name=name, email=email)
+
+        if not instance._id or "mailto:None" in instance._id:
```

Comment:
We are doing this check multiple times. Maybe extracting it into a function makes more sense?